shell/platform/android/io/flutter/plugin/platform/PlatformViewFactory.java

```diff
@@ -25,9 +27,11 @@ public PlatformViewFactory(MessageCodec<Object> createArgsCodec) {
    *     createArgsCodec argument passed to the constructor. This is null if createArgsCodec was
    *     null, or no arguments were sent from the Flutter app.
    */
-  public abstract PlatformView create(Context context, int viewId, Object args);
+  @NonNull
+  public abstract PlatformView create(@Nullable Context context, int viewId, @Nullable Object args);
```

Comment:
Is there a reason why the context is nullable here?

Reply:
Can someone please elaborate here? The context is usually required as NonNull when creating and Android view. I would like to know what is the reason to make this change here.